src/net/mod.rs

```diff
@@ -22,8 +22,11 @@ pub use self::peer::{Acceptor, BootstrapAcceptError, BootstrapAcceptor, Bootstra
                      SingleConnectionError, Uid, bootstrap};
 pub use self::service_discovery::ServiceDiscovery;
 pub use self::socket::{MAX_PAYLOAD_SIZE, Priority, Socket, SocketError};
+pub use self::protocol_agnostic::{PaAddr, PaStream, PaListener};
```

Comment:
Isn't `Pa` prefix redundant, since all of them (`PaAddr`, `PaStream`, etc.) belong to the same module `protocol_agnostic`?

Reply:
Yeah but then they get used in other places throughout crust I didn't want to have to write `protocol_agnostic::` everywhere. Also this naming is consistent with `TcpStream`, `TcpListener`, `UtpStream`, `UtpListener` etc.

Reply:
true